core/components/organisms/pageHeader/__stories__/withNavigationL1.story.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Navigation, Breadcrumbs, Badge, MetaList, PageHeader } from '@/index';
+import { action } from '@storybook/addon-actions';
+
+export const withNavigation = () => {
+  const navigationData = [
+    {
+      name: 'menu_1',
+      label: 'Perormance',
+    },
+    {
+      name: 'menu_2',
+      label: 'Recipients'
+    }
+  ];
+
+  const breadcrumbData = [
+    {
+      label: 'Outreach',
+      link: '/Outreach'
+    }
+  ];
+
+  const navigation = (
+    <Navigation
+      menus={navigationData}
+      onClick={action('menu-clicked')}
+      active={{ name: 'menu_1' }}
+      align="center"
+    />
+  );
+  const actions = (
+    <div style={{ display: 'flex', justifyContent: 'flex-end', alignItems: 'center' }}/>
+  );
+  const breadcrumbs = (
+    <Breadcrumbs
+      list={breadcrumbData}
```

Comment:
We don't need a to declare breadcrumbData in a separate variable as length of data is 1.